modules/consentManagement.js

```diff
@@ -251,6 +402,17 @@ function processCmpData(consentObject, hookConfig) {
   }
 }
 
+function processCcpaData (consentObject, hookConfig) {
+  if (!(consentObject && consentObject.consentString)) {
```

Comment:
there's no `consentString` in the uspapi

Reply:
This is just what I named the value holding the USP string in temp storage. I'll change it to something like `usPrivacy`. However, it should have no impact on the implementation.